paper/paper.bib

```diff
@@ -31,6 +31,44 @@ @article{jones2001scipy
   publisher={https://www.scipy.org}
 }
 
+@article{Hutchinson2001,
+    author = {Hutchinson, J. R. },
+    title = "{Shear Coefficients for Timoshenko Beam Theory }",
+    journal = {Journal of Applied Mechanics},
+    volume = {68},
+    number = {1},
+    pages = {87-92},
+    year = {2000},
+    month = {08},
+    issn = {0021-8936},
+    doi = {10.1115/1.1349417},
+}
+
+@article{Ritto2011,
+  author = {T.G.Ritto and R.H.Lopez and R.Sampaio and J.E.Souza de Cursi},
+  title = {Robust optimization of a flexible rotor-bearing system using the Campbell diagram},
+  journal = {Engineering Optimization},
+  volume = {43},
+  number = {1},
+  pages = {77-96},
+  year  = {2011},
+  publisher = {Taylor & Francis},
+  doi = {10.1080/03052151003759125},
+}
+
+@article{jolly2018,
+  author = {Jolly, Pascal and Arghir, Mihaï and Bonneau, Olivier and Hassini, Mohamed-Amine},
+  title = "{Experimental and Theoretical Rotordynamic Coefficients of Smooth and Round-Hole Pattern Water-Fed Annular Seals}",
+  journal = {Journal of Engineering for Gas Turbines and Power},
+  volume = {140},
+  number = {11},
+  year = {2018},
+  month = {07},
+  issn = {0742-4795},
+  doi = {10.1115/1.4040177},
+  note = {112501},
+}
+
```

Comment:
Add:
@article{timbo2019impact,
  title={Impact of damper seal coefficients uncertainties in rotor dynamics},
  author={Timb{\'o}, Raphael and Ritto, Thiago G},
  journal={Journal of the Brazilian Society of Mechanical Sciences and Engineering},
  volume={41},
  number={4},
  pages={165},
  year={2019},
  publisher={Springer}
  doi={10.1007/s40430-019-1652-8},
}